tests/ssl-opt.sh

```diff
@@ -525,6 +525,46 @@ check_server_hello_time() {
     fi
 }
 
+# Get handshake memory usage from server or client output and check if this value 
+# is not higher then maximum given in the argument
+handshake_memory_check() {
+    MAX_MEMORY="$1"
+    OUTPUT_FILE="$2"
+
+    # Get memory usage from patern like "Heap memory usage after handshake: 23112 bytes. In peak was 33112"
```

Comment:
This comment will need to be updated to match the output format changes recommended previously, in addition to the following change:

```diff
- Get memory usage from patern like
+ Get memory usage from a pattern like
```

Reply:
done